index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon | Your Marketing Experts</title>
+  </head>
+  <body>
+    <!--Header with nav bar-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!--Nav bar with page links-->
+      <nav>
+        <ul>
+          <li>
+            <a id="" href="#search-engine-optimization"
+              >Search Engine Optimization</a
+            >
+          </li>
```

Comment:
The links on your navbar are broken. There are empty IDs here, use the IDs to get your links to work. Remember to give them a test as well and see if they work.